src/components/core/Reloader.react.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux'
+import {getReloadHash} from "../../actions/api";
+
+class Reloader extends React.Component {
+    constructor(props) {
+        super(props);
+        if (props.config.hot_reload) {
+            const { interval } = props.config.hot_reload;
+            this.state = {
+                hash: null,
+                interval,
+                disabled: false
+            }
```

Comment:
What if this state is also handled in a reducer? Would there be value in having this state in the store? Perhaps saving the `hash` in the store for example would make it easier to at some point in the future to enable the store to be saved client-side, so that it can be rehydrated upon refreshing.

Reply:
The hash is null at first, it is set in the state from the server on the first hash request so I don't think it needs to be rehydrated.